Terminal.Gui/Core/ContextMenu.cs

```diff
@@ -110,13 +110,13 @@ public void Show ()
 			} else if (ForceMinimumPosToZero && position.Y < 0) {
 				position.Y = 0;
 			}
-
+			
 			menuBar = new MenuBar (new [] { MenuItems }) {
 				X = position.X,
 				Y = position.Y,
 				Width = 0,
 				Height = 0,
-				UseSubMenusSingleFrame = UseSubMenusSingleFrame
+				UseSubMenusSingleFrame = this.UseSubMenusSingleFrame 
```

Comment:
`this` "Name can be simplified.". See https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/ide0003-ide0009.

Reply:
Can you be more clear on what you are saying/suggesting? 

In my opinion, just because a "Name can be simplified" doesn't mean it SHOULD be simplified. I find code like line 119 to be frustrating to parse. The `this.` makes it clear it's not a bug (as I original thought) but intentional.

Reply:
Sorry @tig, I didn't say it was a bug but only inform the VS2022 advise for potential fixes. Forget please, thanks.